schema/StakeholderInfo/README.md

```diff
@@ -0,0 +1,38 @@
+# Stakeholder
+
+The [Stakeholder](#) is the information about the stakeholder, including the [Entity](../entity/README.md) of the stakeholder and profit sharing ratio hint. Also, if the current digital work is a derivative work, then the source of underlying work is also registered as a creation footprint.
+
+## Schema
+
+```json
```

Comment:
The whole idea of JSON-LD is context sensitive, e.g. the `iscn`, `rdfs` and `schema` in this document is to be explained by the `@context` object appeared before, so isolated object like this may lead to misunderstanding.
May either keep the `@context` object, or expand it to full URI (e.g. by https://json-ld.org/playground/)